tallycash/extension
# Stop marking account activity on dapp connection.

The "lastUserActivityOnNetwork" and "lastUserActivityOnAddress" properties are exclusively used to gate pull-based lookups for user assets / activites on networks.  markAccountActivity is responsible for updating these properties - but there is no utility in them being updated if the wallet is not open.  Since we already mark account activity via the "userActivityEncountered" and "newSelectedAccount" events when a user opens the wallet and selects a new account (respectively) there is no need for us to use a background action like connecting to a dapp to mark account activity.

### To Test
Use the application, navigate between dapps, between networks, add accounts, everything should work as before.

Latest build: [extension-builds-2590](https://github.com/tallycash/extension/suites/9320436077/artifacts/437857387) (as of Tue, 15 Nov 2022 17:45:15 GMT).
